### PR TITLE
Reopen: Clarify and rename identity category

### PR DESCRIPTION
**Pull request by lumjjb**
Opening to replace https://github.com/cncf/sig-security/pull/236 due to inactive author.

Original Text:
>Per #110 , proposing initial changes to clarify the Identity category, and rename it to Identity Lifecycle and Access Management.
>Hopefully this makes it clear that it's not just about each service's own entitlements or authn/authz libraries, but really focused on tools which make it easier for all workloads to support authentication, secure use of tokens, manageable entitlements and control the lifecycle and auditability of those.

### CODE REVIEW

**Comment by lumjjb** [comment]
@izgeri Here's the new PR!

File: landscape/categories.md [hunk 1]
```diff
@@ -23,24 +23,43 @@ sub-categories. The remaining text is a description of each sub-category.
...
+      - Know-Your-Customer/Anti-Money-Laundering projects
+      - Anti-fraud systems
+  - _Access control and enforcement engines_
+    - Attribute and policy-based resource authorization engines and libraries
+      - Including general policy engines
+    - Role-based access control engines
+      - Including libraries and standards for common roles
+    - Privileged Access Management
+      - Applying higher levels of control for sensitive actions, including just-in-time access and enhanced logging
+  - _Resource-specific access layers_
```
Comment (lumjjb) [inline landscape/categories.md]: Copying comment over: Could you clarify what you mean by layer? Layer implies some sort of stack.

Seems like we could replace this with Resource-specific access control?

File: landscape/categories.md [hunk 2]
```diff
@@ -23,24 +23,43 @@ sub-categories. The remaining text is a description of each sub-category.
   - _Secret Management_
     - Tools that help to manage, encrypt or distribute secrets and other sensitive information.
 
-- **Identity & Access Control**
-  - _Identity_
-    - SPIFFE, identity providers, OpenID, LDAP, Okta
-  - _Access Controls_
+- **Identity Lifecycle and & Access Management**
```
Comment (izgeri) [inline landscape/categories.md]:
```suggestion
- **Identity Lifecycle & Access Management**
```

also, I'm not sure how to read this. is it
- identity (lifecycle & access) management, or
- (identity lifecycle) & (access management), or
- ??

I _think_ you mean the second option. is this a tell that maybe `identity lifecycle management` should be its own category, and `access management` its own as well?

File: landscape/categories.md [hunk 3]
```diff
@@ -23,24 +23,43 @@ sub-categories. The remaining text is a description of each sub-category.
...
+  - _Identity Provisioning and Lifecycle Management_
+    - Enrollment and rovisioning of identities for individuals
```
Comment (izgeri) [inline landscape/categories.md]:
```suggestion
    - Enrollment and versioning of identities for individuals
```

not sure this is what was meant here, but taking a stab at the typo

Comment (izgeri) [reply landscape/categories.md]: haha maybe it's just `provisioning`

File: landscape/categories.md [hunk 4]
```diff
@@ -23,24 +23,43 @@ sub-categories. The remaining text is a description of each sub-category.
...
+      - Lifecycle management of identities and associated roles and entitlements
+      - Includes implementations of protocols like SCIM for cross-domain provisioning
+      - See, e.g. NIST SP 800-63A
+    - Identity provisioning for service and machine identities
+      - Cloud-native identity for ephemeral workloads, e.g. the spiffe project
```
Comment (izgeri) [inline landscape/categories.md]:
```suggestion
      - Cloud-native identity for ephemeral workloads, e.g. the SPIFFE project
```

File: landscape/categories.md [hunk 5]
```diff
@@ -23,24 +23,43 @@ sub-categories. The remaining text is a description of each sub-category.
...
+    - Directory and attribute management
+      - Typically implemented atop protocols like LDAP, or other hierarchical and graph-based stores
+  - _Identity Consumption and Single-sign-on_
+    - Identity federation and single sign-on
+      - Typically supporting OpenID connect and SAML
+    - Anonymization
+    - Identity intelligence
+      - Know-Your-Customer/Anti-Money-Laundering projects
+      - Anti-fraud systems
+  - _Access control and enforcement engines_
```
Comment (izgeri) [inline landscape/categories.md]: this section is missing something about audit logs and privilege revocation

File: landscape/categories.md (hunk 2 above)
Comment (ultrasaurus) [reply landscape/categories.md]: Would "Authorization" be a reasonable replacement for "Access Management" ?

**Comment by VinodAnandan** [comment]
I think Authorization is a subset of Access Management. Access Management
includes Authentication, Authorization, Policies, etc.

File: landscape/categories.md (hunk 2 above)
Comment (izgeri) [reply landscape/categories.md]: I think authorization seems more narrow to me than access management, but I'm not sure why

**Comment by lumjjb** [comment]
Will be obselete from https://github.com/cncf/sig-security/issues/348, closing

**Pull request closed without merging**